mesa/agent.py

```diff
@@ -36,6 +37,81 @@ def step(self) -> None:
     def advance(self) -> None:
         pass
 
+    def forward_backward(self, amount, sign):
```

Comment:
I think it is clearer to specify the names with verbs: `move_forward_or_backward`, `move_forward`, `move_backward`. This conveys that they are actions.